layers/+lang/python/funcs.el

```diff
@@ -63,4 +63,6 @@
   " Disable semantic-idle-summary in Python mode.
 Anaconda provides more useful information but can not do it properly
 when this mode is enabled since the minibuffer is cleared all the time."
-  (semantic-idle-summary-mode 0))
+  (add-hook 'semantic-mode-hook (lambda()
+                                  (semantic-idle-summary-mode 0))
+            ))
```

Comment:
Closing parentheses should not got on their own line.

https://www.gnu.org/software/emacs/manual/html_node/elisp/Coding-Conventions.html